.gitignore

```diff
@@ -7,6 +7,7 @@
 *.swp
 *.swo
 *~
+*.DS_Store
```

Comment:
I'm not really sure we should be adding this, and if it should be in a separate PR/commit.